From 5d0b82bc9bfe01fb31639abd851a08252a0fc8fd Mon Sep 17 00:00:00 2001
From: Julien Thomas <jthomas@zenetys.com>
Date: Thu, 18 Apr 2024 12:35:06 +0200
Subject: [PATCH] Add max() function not available in go 1.20

---
 pkg/yqlib/operator_omit.go | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/pkg/yqlib/operator_omit.go b/pkg/yqlib/operator_omit.go
index 45ad7ce..aabb109 100644
--- a/pkg/yqlib/operator_omit.go
+++ b/pkg/yqlib/operator_omit.go
@@ -5,6 +5,13 @@ import (
 	"strconv"
 )
 
+func max(a, b int) int {
+    if a > b {
+        return a
+    }
+    return b
+}
+
 func omitMap(original *CandidateNode, indices *CandidateNode) *CandidateNode {
 	filteredContent := make([]*CandidateNode, 0, max(0, len(original.Content)-len(indices.Content)*2))
 
-- 
2.21.1

